From: Eduard Sanou <dhole@openmailbox.org>
Date: Thu, 3 Oct 2019 12:48:25 +0200
Subject: Honour SOURCE_DATE_EPOCH in manpages

Replace man pages dates with the timestamp from the latest
debian/changelog entry using the variable SOURCE_DATE_EPOCH to make the
package build reproducibly.

Forwarded: not-needed
---
 configure.ac | 6 +++++-
 1 file changed, 5 insertions(+), 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index 0bd4f65..f938716 100644
--- a/configure.ac
+++ b/configure.ac
@@ -117,7 +117,11 @@ AC_CONFIG_FILES([
 	man/rmcpping.8.pre
         rmcpping/Makefile])
 
-ISODATE=`date -u -r ChangeLog +%Y-%m-%d`
+if test -z "${SOURCE_DATE_EPOCH}"; then
+  ISODATE=`date -u -r ChangeLog +%Y-%m-%d`
+else
+  ISODATE=`date -u -d @$SOURCE_DATE_EPOCH +%Y-%m-%d`
+fi
 AC_SUBST([ISODATE])
 
 # libfreeipmi libtool versioning
